src/main/java/frc/lib/math/Conversions.java

```diff
@@ -5,6 +5,37 @@
  */
 public class Conversions {
 
+    /**
+     * @param wheelRPS Wheel Velocity: (in Rotations per Second)
+     * @param circumference Wheel Circumference: (in Meters)
+     * @return Wheel Velocity: (in Meters per Second)
+     */
+    public static double rotationPerSecondToMetersPerSecond(double wheelRPS, double circumference) {
+        double wheelMPS = wheelRPS * circumference;
+        return wheelMPS;
+    }
+
+    /**
+     * @param wheelMPS Wheel Velocity: (in Meters per Second)
+     * @param circumference Wheel Circumference: (in Meters)
+     * @return Wheel Velocity: (in Rotations per Second)
+     */
+    public static double metersPerSecondToRotationPerSecond(double wheelMPS, double circumference) {
+        double wheelRPS = wheelMPS / circumference;
+        return wheelRPS;
+    }
+
+    /**
+     * @param wheelRotations Wheel Position: (in Rotations)
+     * @param circumference Wheel Circumference: (in Meters)
+     * @return Wheel Distance: (in Meters)
+     */
+    public static double rotationsToMeters(double wheelRotations, double circumference) {
+        double wheelMeters = wheelRotations * circumference;
+        return wheelMeters;
+    }
+
```

Comment:
Instead of doubles, use Angle, Distance.... units library and convert the value in the method instead of in the argument.